romfs/lang/de.json

```diff
@@ -61,19 +61,19 @@
         "sd": {
             "help": {
                 "title": "Hilfe",
-                "desc": "Kopiere NSP, NSZ, XCI, or XCZ Dateien auf deine SD Karte,\nstöbere nach ihnen und wähle die zu installierenden Titel aus und drücke dann \ue0b3."
+                "desc": "Kopiere NSP, NSZ, XCI, oder XCZ Dateien auf deine SD Karte,\nsuche nach ihnen und wähle die zu installierenden Titel aus und drücke dann \ue0b3."
```

Comment:
`Kopiere NSP, NSZ, XCI, oder XCZ Dateien` -> `Kopiere NSP, NSZ, XCI oder XCZ Dateien`
In German you don't use a comma before und or oder (or similar words).

Reply:
Fix it too